sycl/include/sycl/ext/oneapi/matrix/query-types.hpp

```diff
@@ -0,0 +1,46 @@
+//==--------------- query-types.hpp - SYCL matrix --------------*- C++ -*---==//
```

Comment:
@dkhaldi please comment if query-types.hpp is OK name for you, or how it should be renamed

Reply:
The name looks okay. Can it be moved to info along with the runtime query code?

Reply:
The types are part of matrix extension specification, so I think it's better to place them together with other matrix extension implementation, so keep them in sycl/include/sycl/ext/oneapi/matrix folder